analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SetTestConstraintCheck.cs

```diff
@@ -44,5 +45,22 @@ public override ProgramState PreProcess(SymbolicContext context)
                 return context.State;
             }
         }
+
+        public override ProgramState PostProcess(SymbolicContext context)
```

Comment:
_Note to myself:_ This needs a redesign in another PR (there's issue with true~first relation)

Reply:
Done in #5192